lms/envs/common.py

```diff
@@ -1620,6 +1620,8 @@
     'survey',
 
     'lms.djangoapps.lms_xblock',
+
+    'openedx.core.djangoapps.content.course_structures',
```

Comment:
@cpennington, @doctoryes : any objections to making content dir in core/djangoapps?